packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.9
```

Comment:
New features increment the minor version, not the bugfix version (applies throughout the PR).

Reply:
I have fixed it.